src/support.js

```diff
@@ -365,7 +365,7 @@ function loadWebAssemblyModule(binary, flags) {
     tableAlign = Math.pow(2, tableAlign);
     // finalize alignments and verify them
     memoryAlign = Math.max(memoryAlign, STACK_ALIGN); // we at least need stack alignment
-    assert(tableAlign === 1);
+    assert(tableAlign === 1, 'invalid tableAlign ' + tableAlign);
```

Comment:
this increases code size. how about putting the entire assert behind `#if ASSERTIONS`? lgtm with that.